tasks/02_kubernetes_fundamentals/kubernetes_4/k8s/dev/slave_apis_deployments.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
```

Comment:
Nit: Wedug code review od @matined , serwisy i deploymenty dla danego poda powinny byc w jednym pliku yamlowym, bo tak to piszac serwis dla poda musicie skakac miedzy plikami (tego akurat sie domyslam).